hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/Operation.java

```diff
@@ -70,6 +70,7 @@
     static final int BITMASK_CALL_TIMEOUT_64_BIT = 1 << 5;
     static final int BITMASK_SERVICE_NAME_SET = 1 << 6;
     static final int BITMASK_CLIENT_CALL_ID_SET = 1 << 7;
+    static final int BITMASK_OP_SENT_BY_CLIENT = 1 << 8;
```

Comment:
Can we avoid using another flag? Especially since it seems that it is used inconsistently eg someone reading this might think that all operations which are part of client message task execution have `isOpSentByClient == true` which is not the case.

Could we maybe use the existing `callerUuid` field to determine if caller is a cluster member or not? I see it is set here for all partition message tasks: https://github.com/hazelcast/hazelcast/blob/40d555dc1110da9f63fd66f504c83d4382d0384b/hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/AbstractPartitionMessageTask.java#L49

Reply:
@vbekiaris makes sense, i have changed code to use `callerUuid`